### ISSUE TITLE
Setting a GaugeSkin removes Color Settings

### ISSUE THREAD

**Issue by ghost** [HanSolo/Medusa#217]
Hello Gerrit,

When I change the GaugeSkin with: gauge.setSkinType(...); it removes all my Color Settings that I did before.
If I set the Skin when Creating the Gauge: Gauge gauge = GaugeBuilder.create().skinType(curve.getSkinType()).build(); It works perfectly, without changing the Colors.

Any Idea how to fix this error?

**Comment by HanSolo** [comment]
Yep the reason for this is the fact that each skin comes with some default settings which might overwrite custom settings. So you might want to apply the color settings after you have changed the skin type.

**Comment by ghost** [comment]
Sadly that doesnt work for me.
I tried it as my first statement then changed the Color to something else and it just ignores the color change.
It could be a Problem from my Side since its a fairly large project.
I will see if I can try it in a new simple project today.
